src/common.cpp

```diff
@@ -69,6 +70,8 @@ to_postgres_type(int arrow_type)
             return TIMESTAMPOID;
         case arrow::Type::DATE32:
             return DATEOID;
+        case arrow::Type::DECIMAL:
```

Comment:
Shouldn't be `DECIMAL128` and `DECIMAL256` used here similar to `bytes_to_postgres_type`?